test/box-luatest/gh_10847_memtx_sort_data_test.lua

```diff
@@ -0,0 +1,584 @@
+local server = require('luatest.server')
```

Comment:
There are no tests for corrupted sortdata. Don't we want to test it?

Reply:
It would be nice to (done, see the [diff](https://github.com/tarantool/tarantool/compare/5b28f8bfe2e44e09410227efb05e1a80dfc6c1e7..bb410e0ef4e62ca1103553dd047ec80497b532a9#diff-4ee7727d3dfbf63579ba3a5fe4ac44a6aeb5a57c2ae56115febecc4946cbde68)).